packages/replicache-perf/src/benchmarks/replicache.ts

```diff
@@ -245,24 +247,26 @@ export function benchmarkRebase(opts: {
         pullInterval: null,
         pushDelay: 9999,
         mutators: {putMap},
-        // eslint-disable-next-line require-await
-        puller: async () => ({
-          response: {
-            cookie: 1,
-            lastMutationIDChanges: {},
-            patch: [
-              {
-                op: 'put',
-                key: 'pull-done',
-                value: true,
-              },
-            ],
-          },
-          httpRequestInfo: {
-            httpStatusCode: 200,
-            errorMessage: '',
-          },
-        }),
+        puller: async () => {
+          await Promise.resolve(); // Make it properly async as requested
```

Comment:
No. Either use `puller: async () => {` with an eslint-disable or no eslint-disable and use Promise.resolve

Reply:
And make sure it passes the type checker